fl33t/client.py

```diff
@@ -265,26 +266,38 @@ def _request(self, method, url, **kwargs):
 
         params = kwargs.pop('params', None)
 
-        LOGGER.debug('Sending {} request with params: {}'.format(
+        self.logger.debug('Sending {} request with params: {}'.format(
```

Comment:
Typically you want to use the `%s` formatting facilities of the logger, so that the arguments are *only* evaluated if the log event is emitted:

```python
self.logger.debug("Sending %s request with params: %s", method, params)
```